fiscalsim_us/parameters/gov/states/mn/tax/income/deductions/charitable_donation_limit.yaml

```diff
@@ -0,0 +1,11 @@
+description: MN charitable donation deduction limit 
```

Comment:
@mitchellpound. This parameter probably needs a new name or at least a new description. Its value of 0.6 is not a limit it is a fraction of something that represents the limit (probably fraction of some measure of income). You should be explicit what this is.

Reply:
This parameter is actually redundant, so I took it out. I realized Minnesota explicitly uses the same logic as the federal charitable donation deduction, so we can just reuse the federal variable instead.